google_shopping.rb

```diff
@@ -9,7 +9,94 @@
 
 
 #outputs the full object
-puts google_data.inspect
+#puts google_data.inspect
 
 #outputs the first item's title
-puts google_data["items"][0]["product"]["title"]
\ No newline at end of file
+#puts google_data["items"][0]["product"]["title"]
+
+#1.) The kind of results you're are dealing are shopping#products. 
+#Go through the items and find all results that have kind of shopping#product. 
+#How many are there? Where else is this count information stored in the search results?
+
+=begin
+
+x=0
```

Comment:
Instead of using a variable called ```x```, I would recommend something more aptly named, like ```shopping_products```.